lua/telescope/builtin/files.lua

```diff
@@ -410,27 +410,21 @@ end
 
 files.tags = function(opts)
   local tagfiles = opts.ctags_file and { opts.ctags_file } or vim.fn.tagfiles()
+  for i,ctags_file in ipairs(tagfiles) do
+    tagfiles[i] = vim.fn.expand(ctags_file, true)
+  end
   if vim.tbl_isempty(tagfiles) then
     utils.notify("builtin.tags", {
       msg = "No tags file found. Create one with ctags -R",
       level = "ERROR",
     })
     return
   end
-
-  local results = {}
-  for _, ctags_file in ipairs(tagfiles) do
-    for line in Path:new(vim.fn.expand(ctags_file, true)):iter() do
-      results[#results + 1] = line
-    end
-  end
+  opts.entry_maker = utils.get_default(opts.entry_maker, make_entry.gen_from_ctags(opts))
```

Comment:
please dont use `utils.get_default` its already deprecated in current `dev` branch and will be removed prior 0.1 (see also #1938 )

use `vim.F.if_nil`

Reply:
Fixed in https://github.com/nvim-telescope/telescope.nvim/pull/1989/commits/9ff4fe1c5495ca32dec6243f8c15bf46c23fa971